<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>665</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesión</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/logo.jpeg</normaloff>../Resources/logo.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:10px;
background-color:transparent;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="root">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="images">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Resources/iniciar-sesion.png</pixmap>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Nombre de usuario</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPlainTextEdit{
border: 1px solid black;
border-radius: 10px;
padding-left: 50px;

}

QPlainTextEdit:hover {
                border-color: blue;
                border-width: 1px;
            }</string>
     </property>
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>200</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-top: -10px;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Resources/user.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>290</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Resources/password.png</pixmap>
     </property>
    </widget>
    <widget class="QCheckBox" name="remember">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>360</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Mantener la sesión</string>
     </property>
     <property name="text">
      <string>Recordar</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../Resources/approval.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
              color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #f43071, stop:1 #27338e);
                border-radius: 10px;
                
                padding: 8px 16px; /* Agregamos un padding para que el botón tenga un tamaño adecuado */
            }</string>
     </property>
     <property name="text">
      <string>INICIO DE SESIÓN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLogin">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>430</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Iniciar sesión</string>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
                background-color: #f43071;
                border: 1px solid transparent;
                border-radius: 10px;
                color: #f5deed;
                padding: 8px 16px; /* Agregamos un padding para que el botón tenga un tamaño adecuado */
            }
            QPushButton:hover {
                background-color: #ff69b4; /* Cambia el color al pasar el mouse */
                border: 1px solid #ff69b4; /* Cambia el borde al pasar el mouse */
            }</string>
     </property>
     <property name="text">
      <string>INICIAR</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="forgotPassword">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>350</y>
       <width>171</width>
       <height>48</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Reiniciar contraseña</string>
     </property>
     <property name="text">
      <string>Olvidó contraseña?</string>
     </property>
    </widget>
    <widget class="QFrame" name="header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">
    background-color: white;
    border-radius: 15px;


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="buttonExit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Salir</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="buttonMinimize">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Minimizar</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="buttonRegister">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Crear cuenta</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                background-color: #27338e;
                border-radius: 10px;
                color: #f5deed;
                padding-left: 20px;
            }
            QPushButton:hover {
                background-color: #3949ab; /* Cambia el color al pasar el mouse */
            }</string>
      </property>
      <property name="text">
       <string> Registrarse</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="services">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Extensiones</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Servicios</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="aboutUs">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Información del programa</string>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-top: -5px;
</string>
      </property>
      <property name="text">
       <string>Sobre nosotros</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="home">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Volver al inicio</string>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-top: -5px;</string>
      </property>
      <property name="text">
       <string>Inicio</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="timeShow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/reloj.gif</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="setTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string>3:15 PM</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>811</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Continuar con Google</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Resources/google.png</normaloff>../Resources/google.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Continuar con X</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Resources/twitter.png</normaloff>../Resources/twitter.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 1px solid black;
border-radius: 10px;
padding-left: 50px;

}

QLineEdit:hover {
 border-color: blue;
border-width: 1px;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonView">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>290</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/view_off.png</normaloff>../Resources/view_off.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Error con la conexión de base de datos</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resources/error-de-datos.png</normaloff>../Resources/error-de-datos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <zorder>password</zorder>
    <zorder>images</zorder>
    <zorder>userName</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>remember</zorder>
    <zorder>label_4</zorder>
    <zorder>buttonLogin</zorder>
    <zorder>forgotPassword</zorder>
    <zorder>header</zorder>
    <zorder>frame</zorder>
    <zorder>buttonView</zorder>
    <zorder>buttonDB</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
